libs/designsystem/item/src/item.component.html

```diff
@@ -14,5 +14,10 @@
     <kirby-icon name="reorder"></kirby-icon>
   </ion-reorder>
 
-  <kirby-icon *ngIf="disclosure" class="disclosure" slot="end" name="{{ disclosure }}"></kirby-icon>
+  <span class="disclosure" slot="end" *ngIf="disclosure">
```

Comment:
This addition `span` is needed to make the animation look correct. The element with `slot="end"` needs to have a left padding of 4px to be positioned correctly relative to the content to the left. If we do the rotation on the same element that has the left padding, then the axis of the rotation seems off.

Reply:
Nice catch 👌🏻